src/qt/lookupaddressdialog.cpp

```diff
@@ -179,7 +147,7 @@ void LookupAddressDialog::searchAddress()
     address.SetString(searchText); // no null check on searchText required we've already checked it's not empty above
     if (address.IsValid()) //do what?
     {
-        // update top fields
+        // update top fieldsgetUserAvailableMPbalance
```

Comment:
@zathras-crypto: this is a paste-mistake, and can be removed/reverted to `// update top fields`.. :)